kinto/__main__.py

```diff
@@ -105,9 +108,15 @@ def main(args=None):
             )
         elif command == "delete-collection":
             subparser.add_argument(
-                "--bucket", help="The bucket where the collection " "belongs to.", required=True
+                "--bucket",
+                help="The bucket where the collection " "belongs to.",
```

Comment:
Can we merge the two strings?
```suggestion
                help="The bucket where the collection belongs to.",
```